tests/test_steps_1_2_3.py

```
@@ -0,0 +1,185 @@
+    print("[OK] lifespan context manager is configured")
+
+    print("\n4. Testing if server can start (health endpoint)...")
+    client = TestClient(app)
+    try:
+        response = client.get("/health")
+        if response.status_code != 200:
+            print(f"[ERROR] Health endpoint returned {response.status_code}")
+            return False
+        print("[OK] Server can start and respond to requests")
+        print(f"[OK] Health endpoint response: {response.json()}")
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)